lib/features/children/overview/pages/edit_allowance_page.dart

```diff
@@ -56,15 +64,16 @@ class _EditAllowancePageState extends State<EditAllowancePage> {
                       const SizedBox(height: 8),
                       Text(
                         'Which amount should be added to\n'
-                            "your child's wallet each month?",
+                        "your child's wallet each month?",
```

Comment:
I think the only thing missing here is that for the generosity challenge it says "Your child(ren)...", maybe we can add a bool isMultiple which exchanges this text? (or any other solution is fine)